client/state/ui/guided-tours/contexts.js

```diff
@@ -122,7 +122,7 @@ export const isSelectedSitePreviewable = state =>
  * @return {Boolean} True if user can run customizer, false otherwise.
  */
 export const isSelectedSiteCustomizable = state =>
-	isSiteCustomizable( state, getSelectedSiteId( state ) );
+	getSelectedSite( state ) && getSelectedSite( state ).is_customizable;
```

Comment:
I know we memoize `getSite`, but maybe it makes sense to call `getSelectedSite()` just once anyway?